### PR TITLE
Adapt Protobuf Definitions

### LINKED ISSUE #131: Analyse and Create service stubs
There are some crud-related endpoints missing.
1. Analyse and locate these endpoints
2. Create stubs with no-operation "handling" these things
3. Mark them as todo and create issues

### LINKED ISSUE #133: Implement required methods for user-svc
```[tasklist]
### Organization
- [x] `create`
- [x] `read`
- [x] `readByUser`
- [x] `update`
- [x] `delete`
- [x] `addMember` by id, must exists
- [x] `removeMember`
- [x] `inviteMember` by email
```

```[tasklist]
### User
- [x] `readSelf`
- [x] `readPublicProfile`
- [ ] `update`
```

### LINKED ISSUE #134: Implement required methods for task-svc
```[tasklist]
### Task
- [x] `create`
- [x] `read`
- [x] `readByPatient`
- [x] `update`
- [ ] `delete` Should we really allow this?
- [x] `assignUser`
- [x] `unassignUser`
```

```[tasklist]
### Patient
- [x] `create`
- [x] `read`
- [x] `readByBed`
- [x] `readByWard`
- [x] `update`
- [ ] `delete` Should we really allow this?
- [x] `assignBed`
- [x] `unassignBed`
```

```[tasklist]
### Bed `Seperate entity and relations are to-be-discussed`
- [x] `create`
- [x] `read`
- [x] `readByRoom`
- [x] `update`
- [x] `delete`
```

```[tasklist]
### Room
- [x] `create`
- [x] `read`
- [x] `readByWard`
- [x] `update`
- [ ] `delete`
```

```[tasklist]
### Ward
- [x] `create`
- [ ] `read`
- [x] `update`
- [ ] `delete`
```

### PR DESCRIPTION
**Pull request by use-to**
Closes #131
Closes #133 
Closes #134

### CODE REVIEW

**Comment by MaxSchaefer** [comment]
Bildschirm­foto 2023-04-27 um 12 19 52 AM

**Comment by MaxSchaefer** [comment]
@use-to ready for review!

**Comment by use-to** [comment]
> @use-to ready for review!

I actually can't review this.
But I recognized some problems.
I'll fix it and mark @FoseFx as reviewer, okay?

**Comment by MaxSchaefer** [comment]
> > @use-to ready for review!
> 
> I actually can't review this. But I recognized some problems. I'll fix it and mark @FoseFx as reviewer, okay?

Sounds good. Unfortunately, GitHub doesn't realize that it would be useful for you to become a reviewer on this PR. When you are done, just re-request the review from FoseFx and me.

File: proto/services/task_svc/v1/patient_svc.proto [hunk 1]
```diff
@@ -9,43 +9,117 @@ option go_package = "gen/proto/services/task-svc";
...
-	string id = 1; // @gotags: validate:"uuid4"
+    string id = 1; // @gotags: validate:"uuid4"
 }
 
 message GetPatientResponse {
-	string id = 1; // @gotags: validate:"uuid4"
-	string human_readable_identifier = 2; // @gotags: validate:"required"
+    string id = 1; // @gotags: validate:"uuid4"
+    string human_readable_identifier = 2; // @gotags: validate:"required"
+    string notes = 3;
```
Comment (FoseFx) [inline proto/services/task_svc/v1/patient_svc.proto:31]: validate tags are missing in a bunch of places

Comment (MaxSchaefer) [reply proto/services/task_svc/v1/patient_svc.proto]: Validate tags for `string notes = 3;`? This should already be handled by proto. Can you mark your concerns?

**Review by MaxSchaefer — APPROVED** [review]

**Review by FoseFx — APPROVED** [review]

**Pull request merged**